mergify_engine/web/__init__.py

```diff
@@ -329,39 +328,10 @@ async def marketplace_handler(
 
 
 @app.get(
-    "/queues/{installation_id}",  # noqa: FS003
+    "/queues/{owner_id}",  # noqa: FS003
```

Comment:
ok, but think that you're going to break the current usage so you need to fix it first… that's gonna work like that.

Reply:
It's already done in this PR: https://github.com/Mergifyio/mergify.io/pull/1901/files#diff-f92db070948ef8a9b2a01cb58fdd7f0396ecba62fb8a3f2244413fa21b8a5c12R94. I made it a few minutes ago.

Reply:
Is it possible to merge the 2 PR at the same time?

Reply:
Not really.
I mean, yes, in theory, we _could_ press the merge buttons at the same time, *but* you shouldn't work that way. It's not safe, and if anything breaks in the middle, it's going to be a pain to go back.

Ending up with half-working stuff is never a good plan. That's why you should plan to do the change in multiple steps like you did so far. It's a little bit more work, but it's way safer as it can be interrupted at any point in time without breaking anything.